src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -98,91 +94,84 @@ def collect_data(geop_result, geojson):
+    z['Acoef'] = erosion_coeff(ws, z['Grow'])
+    z['PcntET'] = et_adjustment(ws)
+    z['KV'] = kv_coefficient(z['Acoef'])
+    z['WxYrBeg'] = max([w.begyear for w in ws])
+    z['WxYrEnd'] = min([w.endyear for w in ws])
+    z['WxYrs'] = z['WxYrEnd'] - z['WxYrBeg'] + 1
 
     # Data from the County Animals dataset
     livestock_aeu, poultry_aeu = animal_energy_units(geom)
-    z.AEU = livestock_aeu / (area * ACRES_PER_SQM)
-    z.n41j = livestock_aeu
-    z.n41k = poultry_aeu
-    z.n41l = livestock_aeu + poultry_aeu
+    z['AEU'] = livestock_aeu / (area * ACRES_PER_SQM)
+    z['n41j'] = livestock_aeu
+    z['n41k'] = poultry_aeu
+    z['n41l'] = livestock_aeu + poultry_aeu
 
-    z.ManNitr, z.ManPhos = manure_spread(z.AEU)
+    z['ManNitr'], z['ManPhos'] = manure_spread(z['AEU'])
 
     # Data from Streams dataset
-    z.StreamLength = stream_length(geom)      # Meters
-    z.n42b = round(z.StreamLength / 1000, 1)  # Kilometers
+    z['StreamLength'] = stream_length(geom)      # Meters
+    z['n42b'] = round(z['StreamLength'] / 1000, 1)  # Kilometers
 
     # Data from Point Source Discharge dataset
     n_load, p_load, discharge = point_source_discharge(geom, area)
-    z.PointNitr = n_load
-    z.PointPhos = p_load
-    z.PointFlow = discharge
+    z['PointNitr'] = n_load
+    z['PointPhos'] = p_load
+    z['PointFlow'] = discharge
 
     # Data from National Weather dataset
-    temps, prcps = weather_data(ws, z.WxYrBeg, z.WxYrEnd)
-    z.Temp = temps
-    z.Prec = prcps
+    temps, prcps = weather_data(ws, z['WxYrBeg'], z['WxYrEnd'])
+    z['Temp'] = temps
+    z['Prec'] = prcps
 
     # Begin processing geop_result
-    z.AgLength = geop_result['ag_stream_pct'] * z.StreamLength
-    z.UrbLength = z.StreamLength - z.AgLength
-    z.n42 = round(z.AgLength / 1000, 1)
-    z.n46e = geop_result['med_high_urban_stream_pct'] * z.StreamLength / 1000
-    z.n46f = geop_result['low_urban_stream_pct'] * z.StreamLength / 1000
+    z['AgLength'] = geop_result['ag_stream_pct'] * z['StreamLength']
+    z['UrbLength'] = z['StreamLength'] - z['AgLength']
+    z['n42'] = round(z['AgLength'] / 1000, 1)
+    z['n46e'] = (geop_result['med_high_urban_stream_pct'] *
+                 z['StreamLength'] / 1000)
+    z['n46f'] = geop_result['low_urban_stream_pct'] * z['StreamLength'] / 1000
 
-    z.CN = np.array(geop_result['cn'])
-    z.SedPhos = geop_result['sed_phos']
-    z.Area = np.array(geop_result['landuse_pcts']) * area * HECTARES_PER_SQM
+    z['CN'] = np.array(geop_result['cn'])
+    z['SedPhos'] = geop_result['sed_phos']
+    z['Area'] = np.array(geop_result['landuse_pcts']) * area * HECTARES_PER_SQM
 
-    z.NormalSys = normal_sys(z.Area)
+    z['NormalSys'] = normal_sys(z['Area'])
 
-    z.AgSlope3 = (geop_result['ag_slope_3_pct'] * area) * HECTARES_PER_SQM
-    z.AgSlope3To8 = (geop_result['ag_slope_3_8_pct'] * area) * HECTARES_PER_SQM
-    z.n41 = geop_result['n41']
+    z['AgSlope3'] = geop_result['ag_slope_3_pct'] * area * HECTARES_PER_SQM
```

Comment:
Since it would replace one line with another line, I don't think the indirection would be more useful than the abstraction. I think simple multiplication is clear enough, and since we'd have to specify both parameters each time, `area * CONVERSION_FACTOR` is easier than `my_conversion_function(area, CONVERSION_FACTOR)`